lib/index.html

```diff
@@ -0,0 +1,105 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Core Components</title>
+  <link rel="stylesheet" href="https://static.nrk.no/core-css/major/1/core-css.min.css">
+  <link rel="stylesheet" href="https://static.nrk.no/core-fonts/major/2/core-fonts.min.css">
+  <style>
+    html { font: 100%/1.6 'LFT Etica',sans-serif; color: #26292a; scroll-behavior: smooth; -webkit-font-smoothing: antialiased }
+    pre, code { overflow: auto; background: #f0f5fe } pre { padding: 1em }
+    a { color: #0059b3 }
+
+    .doc-menu ul { list-style: none; margin: 0; padding: 0 30px; font-size: 14px; line-height: 30px }
+    .doc-menu ul ul a { overflow: hidden; max-height: 30px; animation: fadeIn .5s backwards }
+    .doc-menu a { display: block; color: inherit; text-decoration: none }
+    .doc-menu img { float: right; width: 16px; height: 30px }
+
+    .doc-main:empty { min-height: 9000px; opacity: 0 } /* Allow remember scroll position */
+    .doc-main { overflow: hidden; padding: 4rem 7vw; transition: 1s }
+    .doc-main p { max-width: 700px; margin-top: 0; font-size: 1.1rem }
+    .doc-main h1 { font-weight: 300; font-size: 3rem; line-height: 1; margin: 0 }
+    .doc-main h2 { font-weight: 300; font-size: 1.5em; line-height: 1.5; margin: .5em 0; max-width: 650px }
+    .doc-main h3 { font-weight: 600; font-size: 12px; margin: 2em 0 .5em; text-transform: uppercase }
+    .doc-main hr { border: 0; border-top: 1px solid #eee; margin: 4rem -7vw; width: calc(100% + 14vw) }
+    .doc-main ul { padding-left: 25px; list-style: url("data:image/svg+xml,%3Csvg width='25' height='10' viewBox='0 0 7 10' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M1 1l4 4-4 4' stroke='%23ccc' stroke-width='2' fill='none'/%3E%3C/svg%3E") }
+    .doc-main td code:only-child { display: block; padding: 1em; margin-right: 1em}
+    .doc-main th { font-weight: 600; font-size: 12px }
+    .doc-main table { width: 100% }
+
+    .doc-demo { background: #f0f5fe; padding: 1rem; margin: 1rem 0 }
+    .doc-demo summary { cursor: pointer; padding: 5px 1rem; font-weight: 600; font-size: 12px; border-top: 2px solid #fff }
+    .doc-demo details { margin: 1rem -1rem -1rem }
+    .doc-demo ul { list-style: none }
+
+    @keyframes fadeIn { from { max-height: 0 } }
+    @media(min-width:700px) {
+      .doc-menu { float: left; box-sizing: border-box; position: sticky; top: 0; padding: 70px 0; width: 270px; min-height: 100vh; border-right: 1px solid #eee }
+    }
+  </style>
+</head>
+<body>
+  <script src="https://static.nrk.no/core-masthead/major/1/core-masthead.html.js"></script>
```

Comment:
Antar vi har masthead for å tydeliggjøre branding, men er det nødvendig med lenkene? Vil de være relevante? Kan vi endre til noe mer relevant (kanskje andre produkter fra Fellesteamet istedenfor)?